tokio-trace/test_static_max_level_features/main.rs

```diff
@@ -0,0 +1,75 @@
+#[macro_use]
+extern crate log;
+
+use std::sync::{Arc, Mutex};
+use log::{Level, LevelFilter, Log, Record, Metadata};
+
+#[cfg(feature = "std")]
+use log::set_boxed_logger;
+#[cfg(not(feature = "std"))]
+fn set_boxed_logger(logger: Box<Log>) -> Result<(), log::SetLoggerError> {
+    unsafe {
+        log::set_logger(&*Box::into_raw(logger))
+    }
+}
+
+struct State {
+    last_log: Mutex<Option<Level>>,
```

Comment:
I'd call this `last_level` instead of `last_log`, since we'll be testing the filtering for `tokio-trace`'s `Span` and `Event` types rather than for log messages.
```suggestion
    last_level: Mutex<Option<Level>>,
```